rules/prefer-math-min-max.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const MESSAGE_ID = 'prefer-math-min-max';
+const messages = {
+	[MESSAGE_ID]: 'Prefer `{{replacement}}` to simplify ternary expressions.',
+};
+
+/**
+@param {import('eslint').Rule.RuleContext} context
+@param {import('estree').ConditionalExpression} node
+@param {string} method
+*/
+function reportPreferMathMinOrMax(context, node, left, right, method) {
+	const {sourceCode} = context;
+
+	context.report({
+		node,
+		messageId: MESSAGE_ID,
+		data: {
+			replacement: `${method}()`,
+			value: sourceCode.getText(node),
```

Comment:
There is no `value` placeholder in message.

Reply:
7948804fc2d6d905ce92427ea8f2a3983d70ea1a